calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+# mapping_based_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/mapping_based_calibrator.md)
+
+## Purpose
+
+The package `mapping_based_calibrator` allows extrinsic calibration between 3d lidar and 3d lidar sensors, as well as baselink and 3d lidar sensors used in autonomous driving and robotics.
+
+## Inner-workings / Algorithms
+
+### lidar-lidar calibration
+
+The calibrator is designed to estimate the transformation between lidar and lidar sensors. It starts by employing a lidar, called the `mapping lidar`, for mapping purposes. Then, it utilizes pointcloud registration algorithms to calibrate the remaining lidars, known as `calibration lidars`, to the `mapping lidar`. Specifically, the calibration process encompasses three primary steps: constructing a map, preparing calibration data, and finally estimating the transformations.
+
+A prerequisite for this calibrator is an environment rich in natural landmarks suitable for registration-based mapping in all directions, ensuring that the lidar captures sufficient details beyond simple features like lane surfaces or walls.
```

Comment:
For mapping, sufficient lane surfaces and walls would suffice. This is not well explained. I wrote it in another comment, but the criteria for mapping and calibration are different. Both should be explained to different degrees in all the relevant parts